### ISSUE TITLE
Cannot find module './vendor/ceibo/index.js'

### ISSUE THREAD

**Issue by hergaiety** [embroider-build/embroider#563]
Hey, brand new to Emberoider just following the basic getting started installation steps. I'm seeing this error regardless of how much of our ember-cli-build.js I comment out to narrow it down.

```
⠏ building... [SassCompiler]Building into /tmp/embroider/f5e48f
Build Error (WaitForTrees)

Cannot find module './vendor/ceibo/index.js' from '/tmp/embroider/f5e48f/node_modules/@embroider/synthesized-vendor'


Stack Trace and Error Report: /tmp/error.dump.d9374aa702aeca563a507908f5b50fa9.log
```

In our `yarn.lock` the only reference I can find to `ceibo` is in a dependency of a dependency of a dependency for ultimately `ember-table` (which we use at the latest stable 2.x release)

Log: [error.dump.d9374aa702aeca563a507908f5b50fa9.log](https://github.com/embroider-build/embroider/files/5334611/error.dump.d9374aa702aeca563a507908f5b50fa9.log)

I usually like to provide more information for GitHub issues, but I'm not familiar enough with Embroider or webpack to debug this further. Appreciate any suggestions~

**Comment by ef4** [comment]
Hi, thanks for testing. I tried to quickly reproduce by adding ember-table to a new app (including rendering the table and using the provided test-support code), but that all worked fine.

I see `vendor/ceibo/index.js` is getting included by `ember-cli-page-object` which uses `ember-cli-node-assets` to load it. I have had [other bug reports](https://github.com/embroider-build/embroider/issues/500) about `ember-cli-node-assets` not working under embroider, but I have been unable to reproduce them.

What versions these packages are present in your app?
 - `@embroider/*`
 - `ember-cli-node-assets`
 - `ember-cli`
 - `ember-cli-page-object`

To poke around more, look in `/tmp/embroider/f5e48f`. That is where Embroider is outputting your app and addons after converting them to the v2 format, and before they get combined by webpack. The way I would expect `vendor/ceibo/index.js` to get handled is that it will end up in the `/tmp/embroider/f5e48f/node_modules/@embroider/synthesized-vendor` package. That package is a dumping ground for all the shared vendor files injected by addons. 

`/tmp/embroider/f5e48f/node_modules/@embroider/synthesized-vendor` should list the file under `implicit-test-scripts`, and the actual file should be at `/tmp/embroider/f5e48f/node_modules/@embroider/synthesized-vendor/vendor/ceibo/index.js`. Maybe it's present but at a slightly different path?

**Comment by hergaiety** [comment]
@ef4 Thank you for the well thought out response!

We're running these versions:
```js
    "@embroider/compat": "^0.27.0",
    "@embroider/core": "^0.27.0",
    "@embroider/webpack": "^0.27.0",
    "ember-cli": "~3.19.0",
```

We do not use `ember-cli-page-object` directly, however sub-dependencies do and so the version we ultimately run is `ember-cli-page-object`@`1.16.1` and `ember-cli-node-assets`@`0.2.2`.

Looking in the `synthesized-vendor/package.json` as you've described I do see `"./vendor/ceibo/index.js",` listed under `implicit-test-scripts`. **However, I don't see a ceibo package in the vendor directory here** (though I see plenty of other vendor addons in there). I **do** see a ceibo package under `/tmp/embroider/f5e48f/node_modules/ceibo`, however.

I played around with changing where it looks for the directory but the package.json gets overridden on next build. Similarly, I've tried copying the `ceibo` package into the expected directory but it's lost as soon as I try to build again.
